Plugin/Source/GUI/Assets/gui.xml

```diff
@@ -1,246 +1,260 @@
+                  caption-placement="top-left"/>
+          <Slider caption="Speed [ips]" parameter="speed" slider-type="linear-horizontal"
+                  class="Slider" padding="0" slider-background="ff595c6b" slider-track="ff9cbcbd"
+                  name="Speed" tooltip="Sets the speed of the tape as it affects the playhead loss effects. Note that this control does not affect the wow/flutter processing."
+                  caption-placement="top-left"/>
+          <View flex-grow="0.53" margin="0" padding="2" background-color="00000000">
+            <TextButton margin="0" padding="2" text="3.75" button-color="00000000" background-color="00000000"
+                        onClick="set_speed_3.75" lookAndFeel="SpeedButtonLNF" button-on-color="00000000"
+                        name="3.75 ips" tooltip="Snaps the tape speed to 3.75 inches per second."/>
+            <TextButton text="7.5" margin="0" padding="2" button-color="00000000" background-color="00000000"
+                        onClick="set_speed_7.50" lookAndFeel="SpeedButtonLNF" button-on-color="00000000"
+                        name="7.5 ips" tooltip="Snaps the tape speed to 7.5 inches per second."/>
+            <TextButton margin="0" padding="2" text="15" button-color="00000000" button-on-color="00000000"
+                        background-color="00000000" onClick="set_speed_15.00" lookAndFeel="SpeedButtonLNF"
+                        name="15 ips" tooltip="Snaps the tape speed to 15 inches per second."/>
+            <TextButton margin="0" padding="2" background-color="00000000" onClick="set_speed_30.00"
+                        lookAndFeel="SpeedButtonLNF" text="30" button-color="00000000"
+                        button-on-color="00000000" name="30 ips" tooltip="Snaps the tape speed to 30 inches per second."/>
+          </View>
+          <View flex-grow="0.01" background-color="00000000"/>
+          <PowerButton margin="0" padding="0" background-color="00000000" max-height="25"
+                       min-height="20" button-color="ff595c6b" button-on-color="FFEAA92C"
+                       parameter="loss_onoff" name="Loss On/Off" tooltip="Turns the loss filters on or off."/>
+        </View>
+        <View tab-caption="Degrade" padding="0" flex-direction="column" background-color="FF31323A"
+              margin="0">
+          <View flex-direction="column" max-height="140" background-color="FF1E1F22"
+                flex-grow="2.0" flex-shrink="2.0" padding="0">
+            <View max-height="5"/>
+            <Slider parameter="deg_depth" caption="Depth" class="Slider" name="Depth"
+                    tooltip="Sets the depth of the tape degradation." slider-type="linear-horizontal"
+                    max-height="70" margin="0" min-height="0" slidertext-width="80"
+                    flex-grow="2.5" flex-shrink="2.5" padding=""/>
+            <TextButton flex-align-self="auto" max-height="160" background-color="FF31323A"

```

Comment:
This item needs the `parameter` field to be set to correctly link it with the parameter it's controlling. I think for this case it should `parameter="deg_point1x"`.